crossfire/__init__.py

```diff
@@ -39,3 +42,18 @@ def occurrences(
         max_parallel_requests=max_parallel_requests,
         format=format,
     )
+
+
+def flatten(data, nested_columns=None):
+    nested_columns = set(nested_columns or NESTED_COLUMNS)
+    if not nested_columns.issubset(NESTED_COLUMNS):
+        raise NestedColumnError(nested_columns)
+    if isinstance(data, list):
+        if not data:
+            return data
```

Comment:
This is just to simplify the readability:

```suggestion
    if not data:
        return data
    if isinstance(data, list):
```